shared/MAPL_Throw.F90

```diff
@@ -56,10 +56,11 @@ subroutine MAPL_Fail(filename, line, message)
       character(3) :: prefix
       character(:), allocatable :: base_name
       
-      integer :: rank, ierror
+      integer :: rank, ierror, rc
```

Comment:
Yes - this file should be reverted.   `_VERIFY` indirectly calls throw, so cannot use it here ...